packages/edit-post/src/components/visual-editor/style.scss

```diff
@@ -35,7 +35,7 @@
 }
 
 // The base width of blocks
-.edit-post-visual-editor .block-editor-block-list__block {
+.edit-post-visual-editor .block-editor-block-list__layout-root > .block-editor-block-list__block {
```

Comment:
This is an interesting approach. I was about to suggest that the automatic left and right margin is the exception to the rule, i.e. something that is unique to post content, and even the whole structure of how wide and fullwide images are handled — in fact there's a max-width too.

And so this, in a way, does treat it as the exception, but it does raise the question: is this post-content only, or could we imagine other containers in an FSE future that would want this treatment?